lib/presentation/utils/from_key_to_string.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+import 'package:marvelapp_flutter/Localization/app_localizations.dart';
+
+extension FromKeyToString on BuildContext {
```

Comment:
This can be placed in localization package, BTW package name should be started from the small letter

Reply:
i dont understand. What is BTW and where i wrote form Large letter?

Reply:
done